make.js

```diff
@@ -141,9 +135,17 @@ var MOCHA_OPTS = ' --timeout 15000 --slow 500';
                     target.stop();
                 } catch (e) { }
 
-                if (err) {
+                echo('+ bootlint ' + outputs.join('\\\n\t'));
+
+                // disabling version error's until bootswatch is updated to 3.3.4
```

Comment:
Maybe change this comment to
```
// disable the version check error, since Bootswatch isn't always on the same Bootstrap version
```

Or something like that.